README.md

```diff
@@ -75,6 +75,16 @@ To parse the command line do:
 auto result = options.parse(argc, argv);
 ```
 
+Or if you have all the arguments of `argv` in a `std::string`, separated by
```

Comment:
I think this is a legit concern, though it goes beyond the scope of the problem this patch is trying to target at. Also unlike the single string case, I cannot think of a use case where you are given all the arguments in a single `char*`. I don't see a trivial fix for this to avoid the extra copy of the whole string, but I am happy to learn more on it and address it when possible.

Reply:
@huangzonghao : If you use a class/struct which can be implicitly constructed both from an std::string and from a char* - then you won't need the copy. That class is string_view... see my comment below.